launcher-window/local-page/local-page.js

```diff
@@ -1,12 +1,36 @@
 function launchDockingStation(){
   var { spawn } = require('child_process');
-  var executablePath = "C:/Program Files (x86)/GOG Galaxy/Games/Creatures Exodus/Docking Station/";
+  var executablePath = FindDSPath();
```

Comment:
What if the executable is in say, `C:\DS\` ? Maybe this should be a prompt to locate the DS\C3 folder?

Reply:
@robinduckett is right I think. We should prompt user for ds/c3 location if it can’t be found. And probably save the result to settings for next time.